topics/admin/tutorials/poll-ssa/slides.html

```diff
@@ -0,0 +1,433 @@
+---
+layout: tutorial_slides
+logo: assets/images/gat.png
+video: true
+
+title: "Poll Results: Admin (Small-Scale) Time Burden and Technology Usage"
+
+contributions:
+  authorship:
+  - vladvisan
+  data:
+  - bernt-matthias
+  - lldelisle
+  - hrhotz
+  editing:
+  - hexylena
+
+---
+
+### Context
+
+.center[
+- Poll that took place in February 2024
+- 9 respondents from the [Galaxy SSA group](https://galaxyproject.org/community/sig/small-scale-admins/)
+- Questions about various time burdens and technological choices
+- [Full version of the report, + form export](https://hal.science/hal-04491929) 
+- [Raw data (anonymised) + analysis script](https://entrepot.recherche.data.gouv.fr/dataset.xhtml?persistentId=doi:10.57745/SQMQP1)
+]
+
+---
+
+
+### Active Users
+
+
+.pull-left[![Active Users](../../images/poll-ssa/01-Active-Users.jpg)]
+
+
+.pull-right[
+- 3 categories : <10, circa 20 & circa 50
+- Active vs signed up: no problem having a lot more signed up than active users, no need to delete inactive users (in case they come back)
+]
+
+---
+
+
+### DB Tech
+
+
+.pull-left[![DB Tech](../../images/poll-ssa/02-DB-Tech.png)]
+
+
+.pull-right[
+- Necessarily PostgreSQL
+]
+
+
+---
+
+
+### Computing Method
+
+
+.pull-left[![Computing Method](../../images/poll-ssa/03-Computing-Method.png)]
+
+
+.pull-right[
+- Multiple batch schedulers (usually HTCondor), sometimes with Pulsar, at least one on a separate machine from the batch scheduler, at least one on the same.
+]
+
+
+---
+
+
+### Object Store Backend
+
+
+.pull-left[![Object Store Backend](../../images/poll-ssa/04-Object-Store-Backend.png)]
+
+
+.pull-right[
+- Usually local &/or NAS, a S3.
+]
+
+
+---
+
+
+### Gravity Usage
+
+
+.pull-left[![Gravity Usage](../../images/poll-ssa/05-Gravity-Usage.png)]
+
+
+.pull-right[
+- About 50/50
+- Recent versions seem to include it implicitly
```

Comment:
Good point. I had read the term "Gravity" and that's why I asked the question.

So the slide might still be useful to prospective admins, to realise that Gravity is included, in case they happen upon the term like me.

But the chart itself isn't useful, I agree.

Reply:
Removed https://github.com/galaxyproject/training-material/pull/4972/commits/cafc71cd26d6687091b63a0ec4e10881be3aa65c